scene/main/viewport.cpp

```diff
@@ -3639,6 +3639,7 @@ void Viewport::subwindow_set_popup_safe_rect(Window *p_window, const Rect2i &p_r
 
 	SubWindow sw = gui.sub_windows[index];
 	sw.parent_safe_rect = p_rect;
+	gui.sub_windows.set(index, sw);
```

Comment:
Why not:
```suggestion
	SubWindow &sw = gui.sub_windows.write[index];
	sw.parent_safe_rect = p_rect;
```

Reply:
Because I was not aware of `Vector::write`.